README.md

```diff
@@ -1 +1,257 @@
-# Cryptocurrency-App
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="https://github.com/microverseinc/readme-template/blob/master/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+  # 📖 [Cryptocurrency Hub] <a name="about-project"></a>
+
+> This app is a cryptocurrency price tracker. It was created with redux and react for the fronend and consumed data from coingecko's api to render information to the page
+
+## 🛠 Built With <a name="built-with">React and Redux</a>
+
+### Tech Stack <a name="tech-stack">Frontend</a>
+
+<details>
+  <summary>Client</summary>
+  <summary>Api</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Full info about each token]**
+- **[Live price of each token]**
+- **[Loading and error message incase of any badconnection form api]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Live Video -->
+
+## <a href="https://drive.google.com/file/d/1xI4gOsUqnEPBIZ5xf0MjBO8W5vagiKZ3/view?usp=drivesdk">Live Video</a>
+
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/205274271-926b160b-2bc1-4af7-99ff-2a1e97c0a7f2.PNG)

- [ ] Its great that you have recorded a video but when i click on this link to open your video, it shows me that error shown in the image above. Kindly make this video accessible for all viewers 👍